datasource/models/wikidata/wikidata.py

```diff
@@ -53,17 +52,14 @@ def only_allowed(val):
         df["parent.value"] = df["parent.value"].apply(only_allowed)
 
         # cycle through banks and add them, temporarily ignoring parent relationships
-        existing_tags = {x.tag for x in cls.objects.all()}
         banks = []
         num_created = 0
         bank_values = set(df["bank.value"])
         for bank_value in bank_values:
             bank_df = df[df["bank.value"] == bank_value]
 
             try:
-                num_created, existing_tags = cls._maybe_create_individual_instance(
-                    existing_tags, banks, num_created, bank_df
-                )
+                num_created = cls._maybe_create_individual_instance(banks, num_created, bank_df)
```

Comment:
ok. yeah, I can see how not dealing with tags makes life way easier. Looks like you forgot to update the tests for this though.

Reply:
fixed tests